internal/condition/user_condition_test.go

```diff
@@ -0,0 +1,30 @@
+package condition
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/fabric8-services/fabric8-jenkins-idler/internal/testutils/common"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_get_proxy_response(t *testing.T) {
+	// TODO(chmouel): Need to get the actual response from the proxy, built this
+	// json from reading proxy code.
+	tenantData, err := ioutil.ReadFile("../testutils/testdata/proxy.json")
```

Comment:
Does it not make more sense to keep the test json directly as string in the test? It would make it much easier to take everything in. Now one has to find and open first another file. External files make imo more sense when the data is big and unwieldy to handle otherwise. But in this case it is just such a small piece of JSON.

Reply:
this is how it's done for the others and it's how it's usually done for tests fixtures, usually you would have a mechanic to build those fixtures automatically to make sure the fixtures are in sync with the actual implementation. I would say we should keep it like this,

Reply:
> this is how it's done for the others 

Sure, but as with everything there is a trade-off. Just because it's done so before, does not mean one can change how it is done. Or that there could be multiple ways. But anyways. Let's stick to a response loaded from file then.

> and it's how it's usually done for tests fixtures, 

Not sure about 'usually', but it can be done this way.

> I would say we should keep it like this,

Ok